components/alert/alert.css

```diff
@@ -0,0 +1,34 @@
+.alert-success,
+.alert-warning,
+.alert-error {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 1rem;
+  font-size: 1.25rem;
+  font-weight: 500;
+  border: 1px solid gray;
+  width: 40vw;
+  max-width: 30vw;
+  padding: 1rem 2rem;
+  border: none;
+}
+
+.alert-success span,
+.alert-warning span,
+.alert-error span {
+  font-size: 1.5rem;
+  font-weight: 600;
+}
+
+.alert-success {
+  background-color: rgb(81, 218, 81);
+}
+
+.alert-warning {
+  background-color: rgb(241, 241, 111);
+}
+
+.alert-error {
+  background-color: rgb(255, 109, 109);
+}
```

Comment:
Just suggestion. You can use css variables for it.

Reply:
yeah will do the required refactoring.... thanks for reviewing:)